### ISSUE TITLE
**code smell** find a simpler way to reference only the parent a5 checkboxes

### ISSUE THREAD

**Issue by clairedupree** [clairedupree/wprp-appeal-credit-form#23]
```
  <!-- TODO: find a simpler way to reference only the parent a5 checkboxes, input=name? -->
```

https://github.com/clairedupree/wprp-appeal-credit-form/blob/8db620ee7d475347ef5f40dd913972a2b67c70d8/appeal-credit-application.html#L930

**Comment by clairedupree** [comment]
Use the input=name selector? Or more specific class names?

**Comment by clairedupree** [comment]
Client requested change to form structure, no more need for nested checkboxes (f6d13fd0c7b85a2f138e9a7e2a564f5beb2123f6)

**Comment by clairedupree** [comment]
also fixes #22
